explorer/interpreter/resolve_names.cpp

```diff
@@ -568,6 +577,11 @@ auto NameResolver::ResolveNames(Statement& statement,
 auto NameResolver::ResolveNamesImpl(Statement& statement,
                                     StaticScope& enclosing_scope)
     -> ErrorOr<Success> {
+  if (trace_stream_->is_enabled()) {
+    *trace_stream_ << "** resolving stmt `";
+    statement.PrintID(trace_stream_->stream());
+    *trace_stream_ << "` (" << statement.source_loc() << ")\n";
```

Comment:
```suggestion
    *trace_stream_ << "** resolving stmt `"
                   << PrintAsID(statement)
                   << "` (" << statement.source_loc() << ")\n";
```
I think you can write this using `PrintAsID`.

Reply:
There are a few more `PrintID` calls nearby that you can change in the same way.